RCSwitch.h

```diff
@@ -48,10 +48,8 @@
 #include <stdint.h>
 
 
-// At least for the ATTiny X4/X5, receiving has to be disabled due to
-// missing libm depencies (udivmodhi4)
```

Comment:
Hmm, `receiveProtocol` still performs division, and still invokes `udivmodhi4`, at least over here. The fact that you removed this makes me wonder: Is this due to a new libm? Or a different compiler? Or why else does it work now?